cargo-dist/src/tasks.rs

```diff
@@ -1053,11 +1079,40 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         }
     }
 
-    fn add_source_tarball(&mut self, _tag: &str, to_release: ReleaseIdx) {
+    fn add_extra_artifacts(&mut self, to_release: ReleaseIdx) {
         if !self.global_artifacts_enabled() {
             return;
         }
+        let dist_dir = &self.inner.dist_dir.to_owned();
+        let artifacts = &self.inner.extra_artifacts.clone();
```

Comment:
this needs to be pulling the value off of the release now (there's that one function where we grab all the package configs and slap them onto a Release)

Reply:
also do we know how this would behave in e.g. evil-workspace? like does it run N times, or once-per-package, or..?

Reply:
Tested with evil-workspace. We get one per package if you define it globally, but if you define it per-package you just get the one scoped to that package.